EventKit/eventkit-cloud
# adds support to debug pipelines

## Description of Improvement

Adds configuration to allow celery to be debugged.

## How to test

To test follow the documentation to setup the environment without celery running, and then configure the IDE with the needed environment variables.  Things like breakpoints should work and jobs should successfully process.

## Quality Assurance 

The following items have been updated:


:smile:
